app/javascript/mastodon/features/public_timeline/components/column_settings.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ImmutablePropTypes from 'react-immutable-proptypes';
+import { defineMessages, injectIntl, FormattedMessage } from 'react-intl';
+import ColumnCollapsable from '../../../components/column_collapsable';
+import SettingToggle from '../../notifications/components/setting_toggle';
+import SettingText from './setting_text';
+
+const messages = defineMessages({
+  filter_regex: { id: 'public.column_settings.filter_regex', defaultMessage: 'Filter out by regular expressions' },
+  settings: { id: 'public.settings', defaultMessage: 'Column settings' },
+});
+
+class ColumnSettings extends React.PureComponent {
```

Comment:
Thank you for advising. I modified some coes to solve duplication.

-- I moved one of three ```setting_text.js``` files and deleted other two files.
-- I deleted ```public_timeline/components/colomn_settings.js``` and modified ```public_timeline/index.js``` importing ```community_timeline/components/colomn_settings.js```
-- I modified the locale settings. (deleted ```community.~~~``` and ```public.~~~```, and changed ```home.~~~``` into ```column_settings.~~~``` to use in these three column settings.)

However, my additional commitment caused many conflicts... Would you check these?